examples/dreamcast/raylib/tetris/src/blocks/blocks.cpp

```diff
@@ -0,0 +1,99 @@
+#include "block.h"
+#include "../position/position.h"
+#include "../constants/vmuIcons.h"
+
+class LBlock :public Block{
```

Comment:
```suggestion
class LBlock : public Block{
```
slight style inconsistency